walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -199,6 +199,32 @@ public void doubleBookedPeople() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void doubleBookedPeopleCompleteOverlap() {
+    // Have one person, but have them registerd for two events where on completely overlaps the
+    // other.
+    //
+    // Events  :       |-----A-----|
+    //                    |--A--|
+    // Day     : |-----------------------|
+    // Options : |--1--|           |--2--|
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0900AM, DURATION_2_HOUR),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0930AM, DURATION_1_HOUR),
+            Arrays.asList(PERSON_A)));
+
+    MeetingRequest request = new MeetingRequest(Arrays.asList(PERSON_A), DURATION_30_MINUTES);
+
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
+        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0900AM, false),
+            TimeRange.fromStartEnd(TIME_1100AM, TimeRange.END_OF_DAY, true));
```

Comment:
Same as above comments (with regards to the boolean values).

Reply:
Done.